.travis.yml

```diff
@@ -8,13 +8,14 @@ branches:
   only:
   - develop
   - "/^feature\\/.*$/"
+  - "/^fix\\/.*$/"
 jobs:
   include:
     - stage: "Fundamental-react: Lint"
       script: npm run lint
-    - stage: "Fundamental-react: Build"
+    - stage: "Fundamental-react: Test"
       name: "Unit tests"
-      script: "./ci-scripts/unit-tests.sh"
```

Comment:
this script is no longer used. it could be removed.

@InnaAtanasova - looks like this was meant to gate checkin to make sure code coverage was above a threshold, but I am not sure it was working. Can you confirm?